### DISCUSSION TITLE
Sonarr v3

### DISCUSSION THREAD

**Discussion by johnmillz** [sct/overseerr#1126] [Feature Requests]
Considering Sonarr's main branch is still on v2 should the requirement to use v3 be documented somewhere? Lost some time setting overseerr up only to find this reason in a [closed issue](https://github.com/sct/overseerr/issues/1073).

I think a small note in the setup modal itself would have helped a lot.

BTW off-topic: really fantastic work here, both the product itself and the well looked after repo.

**Comment by TheCatLady** [comment]
@johnmillz It is currently noted in our FAQ (https://docs.overseerr.dev/support/faq#i-receive-409-or-400-errors-when-requesting-a-movie-or-tv-series), but I agree we need to add it to the documentation in more places.

That said, Sonarr v3 is now RC, so it shouldn't be too much longer before popular image maintainers like hotio or LSIO change their latest tags to v3.

**Reply by johnmillz** [reply]
Thanks for the reply, I must have overlooked that part of the FAQ.
Fingers crossed v3 is released soon.

**Reply by sct** [reply]
v3 was released today!
